frontend/public/style/_overrides.scss

```diff
@@ -9,147 +9,6 @@ $pf-4-nav-bar-height: 76px; // Height of the PatternFly 4 masthead
   cursor: auto;
 }
 
-tags-input:focus,
```

Comment:
Moving these lines out of this file breaks our convention of putting overrides of third-party css in a single file and results in us having overrides throughout.  I'm not opposed to that, but do favor consistency.

Reply:
Turns out we are not overriding existing 'react-tagsinput`